### ISSUE TITLE
update_path function is not being exported

### ISSUE THREAD

**Issue by djhurio** [InseeFrLab/rjdworkspace#14]
It looks the `update_path` function is not being exported. Maybe the reason is the extra newline between the `@export` and the function definition.

```
#' @export

update_path <- function(ws_path, raw_data_path, param=8, print_log = FALSE){
```

Thanks!

**Comment by AliceTchang** [comment]
I just updated the MAN file: I hope it works now!

**Comment by djhurio** [comment]
Thanks a lot! The `update_path` is being exported now. However, I still have some problems with the package. I am getting the following error in RStudio:

```
Error in fetch(key) : 
  lazy-load database 'C:/Users/MLiberts/Documents/R/R-4.1.2/library/rjdworkspace/help/rjdworkspace.rdb' is corrupt
```

And the help files of the functions does not open. I see the "Internal Server Error" in the Help tab.

![image](https://user-images.githubusercontent.com/744800/145809084-32a475cd-2be3-4e65-b8f0-42f81109266a.png)

I have reinstalled the package.

**Comment by AliceTchang** [comment]
Hm... If you've used the command devtools::install_github("InseeFrLab/rjdworkspace"), can you try to install it from the tar.gz file? (To retrieve said file, download the package (tar.gz) by entering https://github.com/InseeFrLab/rjdworkspace/archive/master.tar.gz in a browser.)

**Comment by djhurio** [comment]
The same. I did clone the repo and did `devtools::check` on the repo. I am getting an error:

```
> workspace1 <- load_workspace("D:/test_metadata/reference.xml")
Error in load_workspace("D:/test_metadata/reference.xml") : 
  The file doesn't exist or isn't a .xml file !
Execution halted
```

I would recommend to update the package code so you do not get any errors on package checking. I am attaching my [00check.log](https://github.com/InseeFrLab/rjdworkspace/files/7706093/00check.log) for the reference.

**Comment by AliceTchang** [comment]
I added a \donttest{} around the code that raised the last error: it should be fine for the x64 version now

**Comment by djhurio** [comment]
Thank you for your efforts. I installed the latest version. Got the same errors. But after restarting R the problem disappeared.

So, the workaround for me is to install the package and to restart the R session. Then I do not see the problems mentioned before. I am not sure were is the problem.

I got the hint from this [SO post](https://stackoverflow.com/questions/39560808/rendering-an-rmd-fails-because-stringr-rdb-is-corrupt). Thanks!
